media-plugins/distrho-ports/distrho-ports-9999.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit git-r3
+
+DESCRIPTION="Linux ports of Distrho plugins."
+HOMEPAGE="http://distrho.sourceforge.net"
+
+EGIT_BRANCH="master"
+EGIT_REPO_URI="https://github.com/DISTRHO/DISTRHO-Ports.git"
+
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="+alsa +jack"
+
+RDEPEND="alsa? ( media-libs/alsa-lib )
+        jack? ( virtual/jack )"
```

Comment:
I don't think these dependencies are enough.
When  I look at the README.md I see at least `OpenGL/Mesa` and `X11` which is missing.

You can use 
```
qlist -e distroh-ports |  grep -ve "'" | xargs scanelf -L -n -q -F "%n #F" |  tr , ' ' | xargs qfile -Cv | sort -u | awk '{print $1}' | uniq | xargs qatom --format  "%{CATEGORY}/%{PN}" | xargs -L1 qdepends --nocolor --name-only --rdepend --query | grep ^virtual  | uniq
```
to find out the `RDEPEND` packages.